frontend/src/lib/components/Cards/InsightCard/InsightCard.tsx

```diff
@@ -635,9 +645,9 @@ function InsightCardInternal(
                                 : undefined
                         }
                     >
-                        {isUsingDashboardQueryTiles &&
-                        dashboardPlacement &&
-                        ![DashboardPlacement.Public, DashboardPlacement.Export].includes(dashboardPlacement) ? (
+                        {isUsingDashboardQueryTiles && exportedAndCached ? (
```

Comment:
so dashboard query tiles have several states

* flag off -> show unsupported
* flag on -> shared/exported and no cached results -> show unsupported
* flag on -> shared/exported and cached results -> show cached results
* flag on -> not shared/exported -> show "live" query results